lte/gateway/c/core/oai/test/spgw_task/test_spgw_state_converter.cpp

```diff
@@ -50,7 +51,33 @@ TEST_F(SPGWStateConverterTest, TestSPGWStateConversion) {
   }
 }
 
-// TODO add a state conversion test for UE context
+TEST_F(SPGWStateConverterTest, TestUEContextConversion) {
```

Comment:
Let's add an additional unit tests that test the content of each `SpgwUeContext.S11BearerContext` against `s_plus_p_gw_eps_bearer_context_information_t` struct inside `state_teid_ht` hashtable